app/helpers/application_helper.rb

```diff
@@ -8,4 +8,12 @@ def user_avatar(user)
       gravatar_image_url(user.email)
     end
   end
+
+  def current_user_friendship_status(friend_id, user_id, confirmed)
+    Friendship.exists?(friend_id: friend_id, user_id: user_id, confirmed: confirmed)
+  end
+
+  def current_user_friendship_find(friend_id, user_id, confirmed)
+    Friendship.find_by(friend_id: friend_id, user_id: user_id, confirmed: confirmed)
+  end
```

Comment:
There should be no logic in views' helpers, those 2 methods are calling the DB and it's a bad practice to implement in the helpers please move them either to the model or to the controller.